clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -3424,6 +3443,26 @@ llvm::DIMacroFile *CGDebugInfo::CreateTempMacroFile(llvm::DIMacroFile *Parent,
   return DBuilder.createTempMacroFile(Parent, Line, FName);
 }
 
+llvm::DILocation *
+CGDebugInfo::CreateTrapFailureMessageFor(llvm::DebugLoc TrapLocation,
+                                         StringRef FailureMsg) {
+  // Create a debug location from `TrapLocation` that adds an artificial inline
+  // frame.
+  const char *Prefix = "__llvm_verbose_trap";
```

Comment:
Could "__llvm_verbose_trap" live in a shared header file? That way LLDB doesn't need to hardcode the string in a frame recognizer implementation. It can just use the string in the header file and that way things stay in sync automatically.

Reply:
Yes, I think it can be in `clang/include/clang//CodeGen/ModuleBuilder.h` or another header in the directory.